src/parser/jobs/smn/modules/Ruin2.js

```diff
@@ -109,29 +116,32 @@ export default class Ruin2 extends Module {
 
 	_onComplete() {
 		const potLossPerR2 = RUIN3_POT - RUIN2_POT
-		const issues = this._issues.length
-		const warnings = this._warnings.length
+		// It would be a waste to not use any stacks of Further Ruin left after the
+		// last weave of the fight, so do not warn about them.
+		const issues = this._issues.filter(cast => cast.ability.guid === ACTIONS.SMN_RUIN_II.id || cast.timestamp < this._lastOgcd.timestamp).length
```

Comment:
nitpicking; could move that filter fn out so you don't need to define the same logic twice